src/main/java/pl/gajoch/layview/graphics3d/VectorProperties.java

```diff
@@ -0,0 +1,30 @@
+package pl.gajoch.layview.graphics3d;
+
+import javafx.scene.paint.Color;
+
+public class VectorProperties {
+    public double tipLen, tipRadius, radius, lenScale;
+    public Color SpecularColor;
+    public int divisions;
+
+    public VectorProperties(double tipLen, double tipRadius, double radius, double lenScale, Color SpecularColor, int divisions) {
```

Comment:
Common prefix in parameters is strong premise to use [Parameter object](http://refactoring.com/catalog/introduceParameterObject.html).